vault/provider.go

```diff
@@ -236,6 +240,10 @@ type Description struct {
 
 var (
 	DataSourceRegistry = map[string]*Description{
+		"vault_address": {
+			Resource:      UpdateSchemaResource(addressDataSource()),
+			PathInventory: []string{MetaPath},
+		},
```

Comment:
I couldn't decode the order of the data sources in the map or web site index, so I put this first where it will alphabetically be.